src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -420,15 +420,15 @@ public FeedbackSessionQuestionsBundle getFeedbackSessionQuestionsForStudent(
         List<FeedbackQuestionAttributes> questions = fqLogic.getFeedbackQuestionsForStudents(feedbackSessionName,
                 courseId);
 
-
+        Set<String> hiddenInstructorEmails = getHiddenInstrutorEmails(courseId);
```

Comment:
@unyoungwax, are we going to do the check if there are questions which has instructors as the recipients?